modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java

```diff
@@ -71,21 +76,59 @@ public IndexEnv(DataConfig dataConfig, SparkConf sparkConf) throws IOException {
     accumuloTempDir = new Path(dataConfig.hdfsTempDir + "/accumulo");
   }
 
-  public void initAccumuloIndexTable() throws AccumuloSecurityException, AccumuloException,
-      TableNotFoundException, TableExistsException {
-    if (conn.tableOperations().exists(dataConfig.accumuloIndexTable)) {
-      conn.tableOperations().delete(dataConfig.accumuloIndexTable);
+  public static SortedSet<Text> getDefaultSplits() {
+    SortedSet<Text> splits = new TreeSet<>();
+    String splitsFn = "src/main/resources/default-splits.txt";
+    try {
+      try (BufferedReader br = new BufferedReader(new FileReader(new File(splitsFn)))) {
```

Comment:
Could use classloader resource methods to read this.   Should work no matter what the CWD is.